tests/unit_tests/test_wave_06.py

```diff
@@ -228,16 +239,21 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    #raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is falsy
+    assert result is False
     # - That tai and jesse's inventories are the correct length
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
     # - That all the correct items are in tai and jesse's inventories
+    assert item_a or item_b or item_c not in jesse.inventory
+    assert item_d or item_e or item_f not in tai.inventory
```

Comment:
These check to make sure the items aren't in the wrong list, but we would still need to check to see if they are in the right list.